libc/cmake/modules/LLVMLibCCompileOptionRules.cmake

```diff
@@ -230,7 +230,7 @@ endfunction()
 function(_get_hermetic_test_compile_options output_var flags)
   _get_compile_options_from_flags(compile_flags ${flags})
   list(APPEND compile_options ${LIBC_COMPILE_OPTIONS_DEFAULT} ${compile_flags}
-       ${flags} -fpie -ffreestanding -fno-exceptions -fno-rtti)
+       ${flags} -fpie -ffreestanding -fno-exceptions -fno-rtti -save-temps)
```

Comment:
Leftover debug (Also you might need to do `-Wl,--save-temps` if you're doing LTO since the backend is run inside the linker in that case and `-Wl` passes args to the linker).